### DISCUSSION TITLE
How to vertically stack buttons in a FAB context?

### DISCUSSION THREAD

**Discussion by wsw70** [quasarframework/quasar#11320] [General - Components / Directives / etc] [answered]
I would like to stack two action buttons in a sticky page context:

```
<q-page-sticky position="bottom-right" :offset="[18, 18]" class="column">
  <q-btn fab icon="search" color="accent" @click="focusOnSearch"/>
  <q-btn fab icon="search" color="accent" @click="focusOnSearch"/>
 </q-page-sticky>
```

This however still gives me side-by-side icons:

![image](https://user-images.githubusercontent.com/1345886/141610888-1091de8b-4bde-47f9-a12a-10efe22dcaef.png)

**How to stack them vertically?**

EDIT: a codepen is available at https://codepen.io/WoJWoJ/pen/XWaxrwX?editors=1010, the result is even weirder but we can start from there (I removed the `class="column no-wrap"` as it did not change anything)

**Comment by pdanpdan** [comment]
add `no-wrap` class where you put `column`

**Reply by wsw70** [reply]
Thanks, but this did not help

**Comment by pdanpdan** [comment]
Make a codepen

**Reply by wsw70** [reply]
Sure, I added a minimal codepen and updated my question

**Comment by pdanpdan** [comment, marked as answer]
Sorry, I'm on my phone now and cannot update the codepen, but the solution is:
Wrap the buttons in a div with class column no-wrap
So it's q-page-sticky > div > 2*q-btn

**Reply by wsw70** [reply]
Thanks for the follow-up. I already tried that before and it does not help (the codepen with the wrapped buttons: https://codepen.io/WoJWoJ/pen/XWaxWNN)

**Comment by pdanpdan** [comment]
In codepen you cannot use self-closing tags
So it must be `<q-btn></q-btn>`

**Reply by wsw70** [reply]
Ahhh, thank you. Yes indeed - when testing on the actual app the wrapping worked. Thanks!
